admins/pageflow/entry.rb

```diff
@@ -33,7 +33,10 @@ module Pageflow
     sidebar :folders, :only => :index do
       text_node(link_to(I18n.t('pageflow.admin.entries.add_folder'), new_admin_folder_path, :class => 'new'))
       grouped_folder_list(Folder.accessible_by(Ability.new(current_user), :read),
-                          :class => authorized?(:manage, Folder) ? 'editable' : nil,
+                          class: if params[:id] &&
+                                    authorized?(:configure_folder_on, resource.account)
+                                   'editable'
+                                 end,
```

Comment:
This probably means that there are no working spec tests for anything that relies on 'editable'. I would have expected this one here from managing_folders_spec.rb to fail:
```ruby
  scenario 'renaming a folder' do
    folder = create(:folder, name: 'A folder')
    Dom::Admin::Page.sign_in_as(:publisher, on: folder.account)

    visit(admin_entries_path)
    Dom::Admin::FolderPanelItem.find_by_name('A folder').edit_link.click
    Dom::Admin::FolderForm.first.submit_with(:name => 'Renamed folder')

    expect(Dom::Admin::FolderPanelItem.find_by_name('Renamed folder')).to be_present
  end
```